website/docs/general/blog-contributing-guide.md

```diff
@@ -91,274 +105,94 @@ keywords:
-- **Release**: Release notes, this is better understood. It should be noted that the release notes inside the blog are polished, while the release notes inside the release are written by developers.
-- **Security**: Security vulnerability notification and methods to bypass security vulnerabilities, currently there are only two articles, very good to identify, generally have CVE-xxxxxxx is it.
-- **Technology**: Technical articles, easily confused with Practical Case. The factors that determine whether an article is a Technology or a Practical Case are: the content of the article and the subject of the description. For example, if the article is about "Running Apache APISIX on xxx platform", then it is a Practical Case; for example, if the article is about technical stuff, "Apache APISIX v.s. Envoy", then it is Technology.
-- **User Case**: User Case, this is also very straightforward. Please tell us how you use Apache APISIX and your feelings about Apache APISIX.
+Each post can have multiple tags. The tags used currently are given below and each post usually fits into one or more of these tags.
 
-We can help authors to use a better category once a pull request is submitted.
+Each post can have more than one tag. The available tags and explanations are as follows. If none of the tags below fits, please leave a comment in your pull request, and we will handle it together. Please note that these tags and rules of applying tags could change over time.
 
-#### How to get the authors.image_url
+- **Community**: Everything related to community, for example, "How to contribute to an open source project without writing code?".
+- **Events**: Related to events, for example, live streams, event previews, meetups and project meetings.
+- **Interview**: For example, Dr. Yang Li interview, Summer of Programming interview.
+- **Practical Case**: Includes best practices to follow. This is easily confused with **Technology**. The content of the article determines which tag the post belongs to. For example, "Running Apache APISIX on xxx platform" would belong to the Practical Case tag and "Apache APISIX vs Envoy" would belong to the Technology tag.
+- **Release**: Tag for release notes. Note that the release notes in blog posts are polished where as inline release notes are written by developers.
+- **Security**: Security vulnerability notifications and methods to bypass security vulnerabilities. Currently there are [five articles](/blog/tags/security/), and they generally have CVE-xxxxxxx in its title.
+- **Technology**: Technical articles. Should not be confused with **Practical Case** (see above).
+- **User Case**: Posts about using Apache APISIX. Tell us how you are using Apache APISIX!
 
-1. Open Chrome or your preferred browser.
-2. Enter the author's GitHub address to access the author's GitHub home page.
+Reviewers will help you find the right tags while reviewing your PR.
+##### Obtaining authors.image_url
```

Comment:
I can't say if this is a better way of getting a GitHub avatar but it would be worth telling. 
Go to your GitHub profile page and add `.png` to the URL e.g. `https://github.com/username.png`, press enter, and voila you got your avatar URL.

Reply:
This sounds easier and is much better.

Reply:
This would mean changing the screenshot and this would be a concern: https://github.com/apache/apisix-website/pull/893/files/4905320f344a2d0efe81abdbbb1aaf1b9fd351ea#r813531766